README.md

```diff
@@ -606,38 +650,36 @@ Follow the below steps to set up Prometheus and view metrics for Ballerina restf
    http://localhost:19090/
 ```
 
-NOTE:  Ballerina will by default have following metrics for HTTP server connector. You can enter following expression in Prometheus UI
+NOTE:  By default, Ballerina will have the following metrics for HTTP server connector. You can enter the following expressions in Prometheus UI:
 -  http_requests_total
 -  http_response_time
 
 
 ### Logging
 
-Ballerina has a log package for logging to the console. You can import ballerina/log package and start logging. The following section will describe how to search, analyze, and visualize logs in real time using Elastic Stack.
+Ballerina has a log package for logging to the console. You can import `ballerina/log` package and start logging. The following section will describe how to search, analyze, and visualize logs in real time using Elastic Stack.
```

Comment:
Minor correction:
"Ballerina has a log package that allows you to log messages to the console. You can import the `ballerina/log` package and start logging. The following section describes how to search, analyze, and visualize logs in real time using Elastic Stack."